cba_settings_userconfig/cba_settings.sqf

```diff
@@ -104,16 +104,16 @@ force acex_field_rations_timeWithoutWater = 2;
 force acex_field_rations_waterSourceActions = 2;
 
 // ACE Fire
-force ace_fire_dropWeapon = 1;
-force ace_fire_enabled = true;
-force ace_fire_enableFlare = false;
-ace_fire_enableScreams = true;
+force force ace_fire_dropWeapon = 1;
+force force ace_fire_enabled = true;
```

Comment:
Since this is a new feature, my suggestion was to either set `ace_fire_dropWeapon = 0` or `ace_fire_enabled = false`, neither was implemented.

Reply:
Good eye, that was a copy paste on my part in the commit. In the OP I suggested:

// ACE Fire
force force ace_fire_dropWeapon = 1;
force force ace_fire_enabled = false;

Going to fix in the next commit.